### PR TITLE
Budget

### PR DESCRIPTION
**Pull request by Denpeer**
the agent recognizes 2 predicates that help to process its current budget.
the agent has 3 different modules corresponding to the state its current budget in respect to its target.
Modules and predicates are tested

User story:
As a developer, I want the virtual human to make sure that it only builds when it has enough money,, so the virtual human can meet its goals.

### CODE REVIEW

**Comment by Denpeer** [comment]
this version uses vhproject_duplicate as map instead of vhproject, this should avoid problems with multiple agents/users connecting at the same time.

File: src/DUWOagent/knowledge.pl [hunk 1]
```diff
@@ -11,13 +10,24 @@
 
 % Get indicator
 %indicator names: "Astand TUDelft", "Bouw DUWO", "Budget DUWO", "Ruimtelijke kwaliteit", "Variatie Woonruimte"
-getIndicator(Name, Weight, CurrentValue, TargetValue) :- 
+getIndicator(Name, Weight, CurrentValue, TargetValue, ID) :-
```
Comment (MaxGroenenboom) [inline src/DUWOagent/knowledge.pl]: Why isn't ID the first parameter here? That would be more straightforward.

Comment (Denpeer) [inline src/DUWOagent/knowledge.pl]: agreed, I'll change it.

**Comment by MaxGroenenboom** [comment]
Looks good, I approve.

**Comment by klikooo** [comment]
I approve too!

**Comment by MarcianoJ** [comment]
Approved!

**Pull request merged**